packit.spec

```diff
@@ -72,6 +72,7 @@ mv %{buildroot}%{_datadir}/bash_completion/completions/packit %{buildroot}%{_dat
 %{_mandir}/man1/packit*.1*
 %dir %{_datadir}/bash-completion/completions
 %{_datadir}/bash-completion/completions/%{real_name}
+%{_bindir}/packitpatch
```

Comment:
Should this be renamed so that it doesn't begin with `packit`? Placed in the bindir, this will interfere with the autocompletion of the `packit` command. Maybe adding an underscore in the front would already suffice.

Reply:
Or creating `packit patch`, but removing `patch` from autocompletion. This would be similar to the `git` plumbing commands (like `git rev-parse`)

Reply:
I really like the underscore suggestion, gonna go for that one - is that okay?

Reply:
Sure.